alerts/geomodel/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
```

Comment:
I don't think we need to have this file filled with contents, it seems a bit weird to have a version and email defined for a part of mozdef like this.